backend/de.metas.deliveryplanning.base/src/main/java/de/metas/deliveryplanning/GenerateIncomingDeliveryPlanningCommand.java

```diff
@@ -43,19 +45,26 @@
 	private final IAttributeSetInstanceBL attributeSetInstanceBL = Services.get(IAttributeSetInstanceBL.class);
 	private final DeliveryPlanningRepository deliveryPlanningRepository;
 
-	@NonNull private final I_M_ReceiptSchedule receiptSchedule;
-	@NonNull private final DeliveryStatusColorPalette colorPalette;
+	@NonNull
+	private final I_M_ReceiptSchedule receiptSchedule;
+	@NonNull
+	private final DeliveryStatusColorPalette colorPalette;
+
+	@NonNull
+	private final DimensionService dimensionService;
```

Comment:
pls keep annotations on the same line

Reply:
![image](https://github.com/metasfresh/metasfresh/assets/1244701/f7dc5258-8a43-4d52-9d9f-15ab988e33d0)